ui/src/features/annotator/annotation-shape.tsx

```diff
@@ -14,19 +16,38 @@ const getFormattedPoints = (points: Point[]): string => {
 export const AnnotationShape = ({ annotation }: AnnotationShapeProps) => {
     const shape = annotation.shape;
     const color = annotation.labels[0].color;
+    const { setSelectedAnnotation, selectedAnnotation } = useAnnotator();
+    const { scale } = useZoom();
+
+    const isSelected = selectedAnnotation?.id === annotation.id;
+
+    const selectedStyles = {
+        strokeWidth: 2 / scale,
```

Comment:
The current zoom level should be exposed as a css variable (I believe the `ZoomProvider` does this)
That way we can avoid having to rely on any zoom hook here:

```suggestion
        strokeWidth: `calc(2 / var(--zoom-level))`
```

Reply:
Updated